models/user.go

```diff
@@ -24,6 +27,31 @@ func NewUser(loginName string, githubUsername string, slackUsername string, slac
 	}
 }
 
+var UserHeaders = []string{
```

Comment:
Please rename as `userHeaders`, because this variable is used in this file / package only.

This can be detected by [golint](https://github.com/golang/lint).